### PR TITLE
Capybara 2.13 support

### PR DESCRIPTION
**Pull request by twalpole**
This includes 3 previous PRs and Updates support and gemspec to allow Capybara 2.13
* Supports all of the Capybara specified keys, except for :separator, with send_keys.
* Supports returning elements from evaluate_script
* Fixes calling within_frame with the same frame twice in a row

### CODE REVIEW

**Comment by twalpole** [comment]
@mhoran @jferris The failure here isn't caused by the PR - would be good to get consideration on this PR to produce parity with the other drivers and allow use of Capybara 2.13 with capybara-webkit

**Comment by mhoran** [comment]
Thanks @twalpole!

File: src/capybara.js [hunk 1]
```diff
@@ -440,9 +473,34 @@ Capybara = {
...
+  _visitedObjects: [],
+  wrapResult: function(arg) {
+    if (this._visitedObjects.indexOf(arg) >= 0) { return '(cyclic structure)'; }
+    if (arg instanceof NodeList) { arg = Array.prototype.slice.call(arg, 0); }
+    if (Array.isArray(arg)) {
+      for(var _j = 0; _j < arg.length; _j++) {
+        arg[_j] = this.wrapResult(arg[_j]);
+      }
+    } else if (arg && arg.nodeType == 1 && arg.tagName) {
+      return {'element-581e-422e-8be1-884c4e116226': this.registerNode(arg)};
```
Comment (AlanFoster) [inline src/capybara.js]: Out of interest; Where does this magic UUID come from? 🤔

Comment (twalpole) [inline src/capybara.js]: @AlanFoster It's "random" - the WebDriver spec uses a similar UUID -https://www.w3.org/TR/webdriver/#elements - to indicate element references in command responses but states that the value should be a UUID that uniquely identifies the element.  capybara-webkit doesn't currently use a UUID value to uniquely identify elements so I went with a different key.

**Pull request merged**